WorkingWithText/Program.cs

```diff
@@ -40,15 +121,39 @@ public static bool IsValidTime(string hyphenNum)
         // Trim off unneeded spaces.
         public static string PascalConverter(string aFewWords)
         {
-        	return default;
+            string oneWord;
+
+            if(string.IsNullOrEmpty(aFewWords))
+            {
+                return aFewWords;
+            }
+            else
+            {
+                oneWord = System.Threading.Thread.CurrentThread.CurrentCulture.TextInfo.ToTitleCase(aFewWords.Trim().ToLower());
+
+                oneWord = oneWord.Replace(" ", string.Empty);
+
+                return oneWord;
+            }
+        	
         }
 
         // 5- Write a method that accepts an English word. Count the number of vowels
         // (a, e, i, o, u) in the word. So, if the user enters "inadequate", the program should
         // return 6.
         public static int VowelCounter(string aWord)
         {
-            return default;
+            int total = 0;
+            
+            for(int i = 0; i < aWord.Length; i++)
+            {
+                if (aWord[i] == 'a' || aWord[i] == 'e'|| aWord[i] == 'i'|| aWord[i] == 'o' || aWord[i] == 'u'||
```

Comment:
Use .Contains instead of this OR statement.

Also toLower (or toUpper) the input so that you only need to worry about one case of vowels instead of both.